fcm_django/apps.py

```diff
@@ -6,4 +6,4 @@
 class FcmDjangoConfig(AppConfig):
     name = "fcm_django"
     verbose_name = SETTINGS["APP_VERBOSE_NAME"]
-    default_auto_field = "django.db.models.BigAutoField"
+    default_auto_field = "django.db.models.AutoField"
```

Comment:
This doesn't solve the issue as for anyone using the default_auto_field setting with a different value, this will still be a problem. This setting needs to be removed and added to the Device model itself

Reply:
if the project will use `DEFAULT_AUTO_FIELD` from `settings.py` and if in `settings.py` will be `DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'` no issues will be occured. 

But if I set in settings `BigAutoField` the issue will be occured again.

Demo:
Commented field `default_auto_field` and in settings.py 

![image](https://user-images.githubusercontent.com/46032838/148548141-67e80740-ab1b-434f-ba33-dff14d739968.png)

And if I change in settings.py to BigAutoField:

![image](https://user-images.githubusercontent.com/46032838/148548516-f5f54a15-0e08-4af8-b557-4b4444d1d374.png)

Your app shouldn't depend from this setting

Reply:
I mean only in the model, so it doesn't use the project wide settings, uses the BigAutoField for the Device model and leaves all the other models to be whatever you have set it in your project with DEFAULT_AUTO_FIELD

Reply:
Got it.
`id = models.BigAutoField(verbose_name="ID", primary_key=True, serialize=False, auto_created=True)`
in Device model
Is it okay?

Reply:
Yup, double check that it doesn't generate any additional migrations though

Reply:
Actually, models.AutoField is more appropriate so it matches the current state and there's no need for additional migrations

Reply:
Checked.  No migrations for apply.  `serialize=False` by default
Rebased to one commit :)